test/TestProject/TestProject.csproj

```diff
@@ -9,6 +9,7 @@
     <ReadCommittedSnapshot>True</ReadCommittedSnapshot>
     <ServiceBrokerOption>EnableBroker</ServiceBrokerOption>
     <PackageProjectUrl>https://github.com/jmezach/MSBuild.Sdk.SqlProj/</PackageProjectUrl>
+	<GenerateCreateScript>True</GenerateCreateScript>
```

Comment:
Indentation is a little bit off ;).

Reply:
Aaah tabs and spaced :) will fix that (and change notepad++ to do spaces!)